### ISSUE TITLE
Add entity_path to log_file_from_path

### ISSUE THREAD

**Issue by EtaLoop** [rerun-io/rerun#8364]
There is `entity_path_prefix` in `log_file_from_path`, but no `entity_path`.

I'm aware that each DataLoader make its own `entity_path`, but it could be more convenient to decide (or not) the entity_path when calling `log_file_from_path`. 

@teh-cmc :
> On a related note: even if we were to add an --entity-path flag, you would still need to fork and modify the dataloader itself in order for it to actually support that flag.

> What` I would like us to have in the future is server-side rewrite rules, so that you could pass e.g. regex-based remappings in log_file(), and have the Rerun database monkey-patch the dataloaded data so that it follows these rules,e.g.:

#8025 

For now, is it possible to temporary transmit entity_path in `log_file_from_path` ?

**Comment by Wumpf** [comment]
I agree with @teh-cmc that extending the dataloader to take entity path arguments won't fix the issue in its entirety: if you fully control the dataloader you can already make use of the entity path prefix using it as _the_ path rather than the prefix. And if you don't then adding a new entity path option won't do anything either. Also, I'd like to avoid making the interface to the individual dataloaders more complex.

Passing path-rewrite rules is a lot less hacky, not much more difficult and generally very useful
